src/depthmap_generation.py

```diff
@@ -418,13 +418,19 @@ def estimatemarigold(image, model, w, h, device):
     tol = 1e-3
     reduction_method = "median"
     merging_max_res = None
+    resize_to_max_res = None
 
     # From Marigold repository run.py
     with torch.no_grad():
+        image = (image * 255).astype(np.uint8)
```

Comment:
We should be able to simplify by removing
` image = (image * 255).astype(np.uint8)`
 and turning line 431
```
rgb = np.transpose(image, (2, 0, 1))  # [H, W, rgb] -> [rgb, H, W]
rgb_norm = rgb / 255.0
```
into
`rgb_norm = np.transpose(image, (2, 0, 1))  # [H, W, rgb] -> [rgb, H, W]`

That way we aren't multiplying and dividing 255.

Reply:
The rationale behind that is that in Marigold they [resize](https://github.com/prs-eth/Marigold/blob/main/run.py#L140) the image, and the input of that function is `PIL.Image`, so I just thought of having the same option. But yeah, maybe then multiplying and dividing in L427?

Reply:
It does look cleaner to resize with a numpy array using something like
`image = cv2.resize(image, (h, w), interpolation=cv2.INTER_CUBIC)`.

Although I'm not sure if there's any subtle differences between cv2 resize and PIL.Image resize.

Great changes overall!

Reply:
I have no strong feeling either way.